src/OpenTelemetry/CHANGELOG.md

```diff
@@ -2,12 +2,8 @@
 
 ## Unreleased
 
-* Publicize required methods to allow for custom implementations of resource
-  detectors.
+* Added support for implementing custom `ResourceDetector`.
   ([2949](https://github.com/open-telemetry/opentelemetry-dotnet/pull/2949/))
```

Comment:
The links format need to be adjusted for consistency, search "1875" in this document and see how it was done.